src/Components/ADT3DSceneBuilder/Internal/VisualRuleForm/Internal/ConditionsCallout/ConditionsCallout.tsx

```diff
@@ -0,0 +1,241 @@
+                    valueRangeType
+                );
+                conditionCalloutDispatch({
+                    type: ConditionCalloutActionType.FORM_CONDITION_VALUES_SET,
+                    payload: { values: rangeValues }
+                });
+            }
+        },
+        [conditionCalloutState.conditionToEdit?.values, valueRangeType]
+    );
+
+    const onActionValueChange = useCallback(
+        (key: ActionItemKey, value: string) => {
+            if (key === 'color') {
+                conditionCalloutDispatch({
+                    type: ConditionCalloutActionType.FORM_CONDITION_COLOR_SET,
+                    payload: { color: value }
+                });
+            } else {
+                conditionCalloutDispatch({
+                    type: ConditionCalloutActionType.FORM_CONDITION_ICON_SET,
+                    payload: { iconName: value }
+                });
+            }
+        },
+        []
+    );
+
+    const handleSaveClick = useCallback(() => {
+        onSave(conditionCalloutState.conditionToEdit);
+        onDismiss();
+    }, [onDismiss, onSave, conditionCalloutState.conditionToEdit]);
+
+    const handleCancelClick = useCallback(() => {
+        onDismiss();
+    }, [onDismiss]);
+
+    // styles
+    const classNames = getClassNames(styles, {
+        theme: useTheme()
+    });
+
+    return (
+        <Callout
+            target={target}
+            styles={classNames.subComponentStyles.callout}
+            onDismiss={onDismiss}
+            directionalHint={DirectionalHint.rightCenter}
+            hidden={calloutType === CalloutInfoType.inactive}
+        >
+            <Stack tokens={{ childrenGap: 12 }}>
+                <h4 className={classNames.title}>
+                    {calloutType === CalloutInfoType.create
+                        ? t(LOC_KEYS.addCondition)
+                        : t(LOC_KEYS.editCondition)}
+                </h4>
+                <TextField
+                    value={
+                        conditionCalloutState.conditionToEdit?.visual
+                            ?.labelExpression
+                            ? conditionCalloutState.conditionToEdit.visual
+                                  ?.labelExpression
+                            : ''
+                    }
+                    onChange={onLabelChange}
+                    label={t(LOC_KEYS.labelField)}
+                    required={true}
+                />
+                <ConditionSummary
+                    onChangeValues={onValuesChange}
+                    currentValues={
+                        conditionCalloutState.conditionToEdit?.values
+                    }
+                    conditionType={valueRangeType}
+                    areValuesValid={validityMap.current.ranges}
+                />
+                <ActionItem
+                    setActionSelectedValue={onActionValueChange}
+                    color={conditionCalloutState.conditionToEdit?.visual.color}
```

Comment:
The default color value seems null:
![image](https://user-images.githubusercontent.com/45217314/198159239-02e78a75-2558-4278-bdd3-eff7350a6349.png)